WordPress/gutenberg
# Patterns: Reinstate template parts mode spec

Related: https://github.com/WordPress/gutenberg/pull/51078#issuecomment-1637834389

## What?

Reinstates tests for template parts pages for classic themes.

## Why?

The spec was removed when the manage all template parts page was removed. It was later reinstated however the spec wasn't.

## How?

Adds the tests from the deleted spec file into the new hybrid-theme spec.

## Testing Instructions

1. Confirm the reinstated tests match the removed file (c4961341cf918d8281997e4cfbbe7ae1b1b48c80) 
2. Run the e2e spec and confirm tests pass: 
`npm run test:e2e:playwright  -- specs/site-editor/hybrid-theme.spec.js`


## Screenshot

<img width="1206" alt="Screenshot 2023-07-20 at 3 29 41 pm" src="https://github.com/WordPress/gutenberg/assets/60436221/ad958e61-d7a2-402c-b43f-d9a9469df454">
